docs/status.md

```diff
@@ -136,9 +136,9 @@ one of the following tracking labels.
 | select_and_scatter       | no            | revisit      | no             | no              | no          |
 | send                     | no            | revisit      | no             | no              | no          |
 | set_dimension_size       | no            | yes*         | yes*           | yes             | no          |
-| shift_left               | no            | yes*         | yes*           | yes             | no          |
-| shift_right_arithmetic   | no            | yes*         | yes*           | yes             | no          |
-| shift_right_logical      | no            | yes*         | yes*           | yes             | no          |
+| shift_left               | yes           | yes          | yes            | yes             | no          |
```

Comment:
Verification for all three would need to be `revisit` here since it does not currently check that only integers are allowed.

Reply:
Sure